app/scripts/views/settings/settings-file-view.js

```diff
@@ -512,10 +512,22 @@ const SettingsFileView = Backbone.View.extend({
                 this.backupInProgress = false;
                 backupButton.text(Locale.setFileBackupNow);
                 if (err) {
+                    let title = ''
```

Comment:
Please add commas. You can also validate violations with eslint to see all of them.

Reply:
Whoops, sorry. Forgot to enable eslint in my IDE. Fixed.